### PR TITLE
Context Binding (Element Binding): Fix code snippets

### PR DESCRIPTION
**Pull request by boghyon**
- Fixed invalid JSON, JS, and XML which contained syntax errors
- Removed trailing whitespace
- Aligned indentations with other snippets

### CODE REVIEW

File: docs/Context_Binding_(Element_Binding)_91f05e8.md [hunk 1]
```diff
@@ -68,37 +67,36 @@ Element binding is especially interesting for containers or layouts containing m
 ``` xml
 <mvc:View
 	controllerName="sap.ui.sample.App"
-	xmlns="sap.m"
-	xmlns:mvc="sap.ui.core.mvc">
-	<l:VerticalLayout id="vLayout"
+	xmlns:mvc="sap.ui.core.mvc"
```
Comment (flovogt) [inline docs/Context_Binding_(Element_Binding)_91f05e8.md:70]: xmlns="sap.m" is missing here

Comment (flovogt) [reply docs/Context_Binding_(Element_Binding)_91f05e8.md:70]: required by sap/m/Text

File: docs/Context_Binding_(Element_Binding)_91f05e8.md [hunk 2]
```diff
@@ -226,16 +229,11 @@ Say we want to display companies in a `sap.m.List` control. Here’s what the XM
...
-	xmlns:mvc="sap.ui.core.mvc">
-	  <List id=”companyList” items="{companies}">
-		<items>
-			<StandardListItem
-	 	title="{name}"
-	 	description="{city}"
-			/>
-		 </items>
-	  </List>
+	xmlns:mvc="sap.ui.core.mvc"
```
Comment (flovogt) [inline docs/Context_Binding_(Element_Binding)_91f05e8.md:232]: @boghyon any reason why you prefer to put the default namespace not as the first namespace?

**Review by flovogt — CHANGES_REQUESTED** [review]

**Review by KlattG — APPROVED** [review]
LGTM

File: docs/Context_Binding_(Element_Binding)_91f05e8.md (hunk 1 above)
Comment (boghyon) [reply docs/Context_Binding_(Element_Binding)_91f05e8.md:70]: Added. 👍 Thanks for the review!

File: docs/Context_Binding_(Element_Binding)_91f05e8.md (hunk 2 above)
Comment (boghyon) [reply docs/Context_Binding_(Element_Binding)_91f05e8.md:233]: I namespaces are added in the order of the respective controls in the XML definition. So, `xmlns:mvc="sap.ui.core.mvc"` happens to be always the first one (`<mvc:View ...>`) and `sap.m` at some point later.

**Comment by boghyon** [comment]
@flovogt The missing `xmlns="sap.m"` is added now in my [2nd commit](https://github.com/SAP/openui5-docs/pull/56/commits/a086af0d3f2414eb5472f340c6d6cf086a107d85).

Not sure what exactly was changed previously in https://github.com/SAP/openui5-docs/commit/c4db0e89c71c4b764f9eeb8c50c016a758a24319#diff-d4957235d36be7557b7c39b060f21c73aa256ad0469b84c81bb616c9664afb73 which caused a merge conflict in this PR. I resolved it with the [3rd commit](https://github.com/SAP/openui5-docs/pull/56/commits/df9afbf62b2cc4e9ef3119e5b520b3670d413679).

**Comment by flovogt** [comment]
@boghyon Now, the whole file content is changed. So the different code blocks you've changed are no longer visible. Could you please have another look on it?

**Comment by boghyon** [comment]
@flovogt Yes, I can see that the changes in the code blocks are still there. I had to resolve the conflict manually by accepting the current content from the master branch then reapplying my changes from my 1st and 2nd commit.

**Review by KlattG — APPROVED** [review]

**Comment by KlattG** [comment]
Fixed from 1.99

**Comment by flovogt** [comment]
Thanks for your contribution @boghyon 🚀

**Pull request merged**